pyfr/solvers/baseadvec/elements.py

```diff
@@ -42,15 +42,15 @@ def set_backend(self, *args, **kwargs):
         }
 
         # Interpolation from elemental points
+        kernels['disu'] = lambda: self._be.kernel(
+            'mul', self.opmat('M0'), self.scal_upts_inb,
+            out=self._scal_fpts
+        )
+
         if fluxaa:
-            kernels['disu'] = lambda: self._be.kernel(
-                'mul', self.opmat('M8'), self.scal_upts_inb,
-                out=self._scal_fqpts
-            )
-        else:
-            kernels['disu'] = lambda: self._be.kernel(
-                'mul', self.opmat('M0'), self.scal_upts_inb,
-                out=self._scal_fpts
+            kernels['uqpts'] = lambda: self._be.kernel(
```

Comment:
See if we can think of a more consistent name; not that the current convention is particularly good.

Reply:
Any suggestions?